01-ipm.Rmd

```diff
@@ -0,0 +1,95 @@
+
+<center>![](./www/ipm_walk4.png)</center>
+
+After you select your Structure, you can move on to the Harvest Tab. Decide your effort variable and the years that you will be conducting your analysis, then look at the given information for the years that will be forecast. You can change the Harvest by double clicking the numbers and typing in new entries.
+
+<center>![](./www/ipm_walk5.png)</center>
+
+Our next step is the Run tab, which again already has a decent set of default values loaded. Adjust your Burnin Length, MCMC iterations, and Thinning Length. Decide whether you would like to automate convergence (This may increase the time it takes to run the model).
+
+<center>![](./www/ipm_walk6.png)</center>
+
+Once you are satisfied with all of the inputs outlined above, click [<b>Fit Model</b>]{style="color: #196eb0;"} to begin analysis. This should take some time, but you will get a dialog box letting you know when the process is complete
+
+### Looking at Data {#ipm-look}
+
+(NOTE: This section will only work if you completed the steps above to prepare your data and fit your model beforehand.)
+
+To look at the data you prepared, wait until the model that you initiated in the setup tab has finished running, then go ahead and click the Plots tab on the IPM dropdown menu to check the results. Hovering your mouse over a point on a plot will give you the parameter, year, age, sex, control limits, and mean associated with the dataset that created the data point.
+
+<center>![](./www/ipm_walk7.png)</center>
+
+You can also check the model output by clicking the Tables tab right below the Plots one on the IPM dropdown. Rhat values greater than 1.1 are highlighted in red, which means the model may not have converged and should be run again after modifying the settings on the Run tab in the Setup tool. The first ten entries that appear when you open the table tab are not the only outputs of the model. You will notice that you may navigate the pages with the list to the bottom right, or you may increase the number of entries visible at the top left of the page, using the [<b>Show (10) Entries</b>]{style="color: #196eb0;"} dropdown.
+
+<center>![](./www/ipm_walk8.png)</center>
+
+As a final step to get more diagnostic information, use the Build Report tool. 
+
+<center>![](./www/ipm_walk10.png)</center>
+
+Simply click on the tool and then click the download prompt on the popup window. The resulting document contains interactive plots as well as model specifications and fit.
+
+<center>![](./www/ipm_walk9.png)</center>
+
+## Reference {#ipm-ref}
+
+### Setup {#ipm-setup}
+
+The main purpose of the Setup tab is to load and preview data that will be used for later analysis, and we will spend more time describing each element in detail. The window on the top left of the page has several tabs related to data preparation labeled Overview, Structure, Harvest, and Run.
```

Comment:
A picture here that shows all of the tabs (Setup, Overview, Structure, Harvest, Run) might help with visualization.